examples/everything.amp.html

```diff
@@ -235,6 +235,31 @@ <h1 id="top">AMP #0</h1>
       Open Lightbox
   </button>
 
+  <h2>Scrollable Lightbox</h2>
+  <amp-lightbox scrollable id="lightbox-scrollable" layout="nodisplay" class="lightbox">
+    <p>
+      <button on="tap:lightbox-scrollable.close">
```

Comment:
are these buttons still black-on-black? if so, let's change their background color to white.